django/db/migrations/executor.py

```diff
@@ -329,8 +329,11 @@ def should_skip_detecting_model(migration, model):
         apps = after_state.apps
         found_create_model_migration = False
         found_add_field_migration = False
+        ignores_table_name_case = self.connection.features.ignores_table_name_case
```

Comment:
Might be worth using a different name since it's also used for column detection. What about

```python
fold_identifier_case = self.connection.features.ignores_table_name_case
```